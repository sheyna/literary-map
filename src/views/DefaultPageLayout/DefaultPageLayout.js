import styles from './DefaultPageLayout.module.css';

function DefaultPageLayout({children}) {
  return (
    <main className={styles.defaultPage}>
      {children}
    </main>
  )
}

export default DefaultPageLayout
