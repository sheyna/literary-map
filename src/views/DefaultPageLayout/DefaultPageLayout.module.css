.defaultPage {
  width: 80%;
  max-width: 30rem;
  margin: 4rem auto;
  min-height: 42rem;
}
