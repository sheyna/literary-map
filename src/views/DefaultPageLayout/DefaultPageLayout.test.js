import ReactTestRenderer from 'react-test-renderer';
import DefaultPageLayout from  './DefaultPageLayout';

describe('DefaultPageLayout component', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(
      <DefaultPageLayout>
        <p>child</p>
      </DefaultPageLayout>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
