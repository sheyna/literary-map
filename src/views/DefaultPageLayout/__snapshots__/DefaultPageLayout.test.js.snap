// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultPageLayout component should match snapshot 1`] = `
<main
  className="defaultPage"
>
  <p>
    child
  </p>
</main>
`;
